src/client/java/minicraft/screen/InventoryMenu.java

```diff
@@ -17,12 +19,17 @@ class InventoryMenu extends ItemListMenu {
 		super(ItemListMenu.getBuilder(entryPos), ItemEntry.useItems(inv.getItems()), title);
 		this.inv = inv;
 		this.holder = holder;
+		this.title = title;
+		this.entryPos = entryPos;
 	}
 
 	InventoryMenu(InventoryMenu model) {
-		super(ItemListMenu.getBuilder(), ItemEntry.useItems(model.inv.getItems()), model.getTitle());
+		super(ItemListMenu.getBuilder(model.entryPos), ItemEntry.useItems(model.inv.getItems()), model.title);
 		this.inv = model.inv;
 		this.holder = model.holder;
+		this.creativeInv = model.creativeInv;
+		this.title = model.title;
+		this.entryPos = model.entryPos;
 		setSelection(model.getSelection());
```

Comment:
This seems unneccecary.

Reply:
The fields are save in this subclass. `title` changes after menu creation. Those values ensure that the configuration does not change after construction copying.

Reply:
I think it’s fine, shouldn’t break anything.